Dockerfile

```diff
@@ -46,6 +79,6 @@ RUN composer dump-autoload --optimize
 RUN composer clear-cache
 
 # Update permisions
-RUN chown -R www-data:www-data /var/www/html/
+RUN chown -R www-data:www-data /opt/drupal/web/
```

Comment:
Now we're explicit about the `WORKDIR`, it feels like we can dry this up in one of two ways which might help keep things consistent in future:

1. Use `.`, here, so we're using the working directory
2. Set an environment variable, and use that to set this and `WORKDIR`